Userland/Libraries/LibJS/Runtime/ObjectPrototype.cpp

```diff
@@ -111,13 +111,13 @@ JS_DEFINE_NATIVE_FUNCTION(ObjectPrototype::value_of)
 
 JS_DEFINE_NATIVE_FUNCTION(ObjectPrototype::property_is_enumerable)
 {
-    auto name = vm.argument(0).to_string(global_object);
+    auto property_key = StringOrSymbol::from_value(global_object, vm.argument(0));
```

Comment:
Oh look, you found what my recent meme was about :D

Sadly this is still not fully spec compliant, as `from_value()` differs from `ToPropertyKey()` in subtle ways - still better than ignoring symbols altogether though, I'll take it :)

Reply:
too bad, ill look into it a bit more tommorow then